modules/ocf_desktop/manifests/packages.pp

```diff
@@ -34,21 +34,39 @@
     ['armagetronad', 'gl-117', 'gnome-games', 'minecraft-launcher', 'redeclipse',
       'wesnoth', 'wesnoth-music']:;
     # graphics/plotting
-    ['r-cran-rgl', 'jupyter-qtconsole', 'rstudio']:;
+    ['r-cran-rgl', 'jupyter-qtconsole']:;
     # input method editors
     ['fcitx', 'fcitx-libpinyin', 'fcitx-rime', 'fcitx-hangul', 'fcitx-mozc']:;
     # nonfree packages
     ['firmware-linux']:;
     # notifications
     ['libnotify-bin', 'notification-daemon']:;
-    # performance improvements
-    ['preload']:;
     # security tools
     ['scdaemon']:;
     # utilities
     ['wakeonlan']:;
     # Xorg
-    ['xclip', 'xsel', 'xserver-xorg', 'xscreensaver', 'freerdp-x11']:;
```

Comment:
I have used rdesktop but personally prefer freerdp. Freerdp2-x11 is available in both stretch backports and buster, so perhaps we can use that?

Reply:
Ah, good idea, yeah that should work ok for both